packages/contracts-bedrock/src/dispute/interfaces/IPermissionedDisputeGame.sol

```diff
@@ -27,6 +28,19 @@ interface IPermissionedDisputeGame is IDisputeGame {
         address counteredBy;
     }
 
+    struct GameConstructorParams {
```

Comment:
Is this still needed here since it's already imported from `IFaultDisputeGame`?

Reply:
Yes it's still needed to satisfy the interface checks

Reply:
it actually passes it on my end without this struct

Reply:
Ah weird, will remove it. I suspect that means we have a bug in our verification logic somewhere since it should not pass if there are unnecessary things in the interface?

Reply:
I think this is because the struct is only defined in the AST and not in the ABI because no function/error uses it. The interface script does not check the AST i guess